1. Mexiko Create filenames filename-mappings and infofiles.py

```diff
@@ -283,16 +283,16 @@ def create_infofiles(row, filenames_file, not_ok_file):
     
     if pd.notnull(row["Personnamn / fotograf"]):
         if "Apenes" in row["Personnamn / fotograf"]:
-            infotext +="|photographer       =  " + "{{creator:Sigvald_Linné}} / Ola Apenes\n"
+            infotext +="|photographer       = " + "{{creator:Sigvald_Linné}} / Ola Apenes\n"
```

Comment:
Since the following check comes afterwards, doesn't it suffice with adding {{creator:Ola_Apenes}}?